src/backend/aspen/api/main.py

```diff
@@ -32,19 +32,40 @@
 
 
 def get_allowed_origins() -> List[str]:
-    allowed_origins = []
-    deployment = os.getenv("DEPLOYMENT_STAGE")
+    """Allowed origins for cross origin requests. See `CORSMiddleware`.
+
+    Gets list of all allowed origins for cross origin requests. This is pretty
+    simple right now: we only serve the FE of our app and Galago with content
+    from our BE, so that's all this needs to cover. For ease, the Galago URLs
+    are just hardcoded. If we ever need to start handling a lot more CORS
+    requests or the Galago URLs become more dynamic, we should reevaluate
+    current process.
+    """
+    GALAGO_URLS = (
```

Comment:
You can make this a literal list with [] and then not have to use list() below.

Reply:
Yeah, that's a good point. I put it in the tuple because I was worried about the Galago URLs somehow getting mutated, but it's within a function, so I'm being silly.